.github/workflows/build-push-docker-images.yml

```
@@ -0,0 +1,136 @@
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+          sparse-checkout: |
+            ${{ inputs.context }}
+          sparse-checkout-cone-mode: true
+
+      - name: Get tags
+        run: git fetch --tags origin
+
+      - name: Get version
+        if: ${{ inputs.image_name }} == 'workflows-service'
```

actionlint (1.7.3)

if: condition "${{ inputs.image_name }} == 'workflows-service'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)